common/app/conf/switches/PrivacySwitches.scala

```diff
@@ -0,0 +1,38 @@
+package conf.switches
+
+import conf.switches.Expiry.never
+import conf.switches.Owner.group
+import conf.switches.SwitchGroup.{Privacy}
+
+trait PrivacySwitches {
+
+  val Cmp = Switch(
+    SwitchGroup.Privacy,
+    "consent-management",
+    "Enable consent management. Individual frameworks will also need to be switched on.",
```

Comment:
What happens to features that need consent when this is turned off? They only work for users who have already consented?

Reply:
the CMP wouldn't initialise, so there would be no privacy management at all. anything that needs consent would not run. 

not sure why we'd need this normally, but there was no way of disabling it when sourcepoint was down on friday, so it's designed to cover that type of scenario